<?php defined('BASEPATH') OR exit('No direct script access allowed'); ?>

ERROR - 2021-01-24 00:37:39 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 04:34:50 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 04:53:23 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 05:37:43 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 06:27:21 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 07:27:04 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 07:34:43 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 08:39:41 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 08:49:03 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 08:55:59 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 10:15:10 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 10:39:06 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 10:54:19 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 11:18:31 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 12:22:06 --> 404 Page Not Found: Consignee/1
ERROR - 2021-01-24 13:37:23 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 16:57:53 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 18:59:13 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 21:45:42 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 21:47:29 --> 404 Page Not Found: Consignee/1
ERROR - 2021-01-24 21:47:30 --> 404 Page Not Found: Consignee/1
ERROR - 2021-01-24 21:49:00 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:00 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:00 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:00 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:01 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:01 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:01 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:02 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:02 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:02 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:02 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:02 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:02 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:02 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:02 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:02 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:03 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:03 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:03 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:03 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:03 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:03 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:03 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:03 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:03 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:03 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:04 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:04 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:04 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:04 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:04 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#2 [internal function]: _exception_handler(Object(Error))
#3 {main}
  thrown D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:05 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:05 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::prepare...', 'D:\\xampp\\htdocs...', 122)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::prepare...', 'D:\\xampp\\htdocs...', 122)
#2 [internal function]: _error_handler(2, 'mysqli::prepare...', 'D:\\xampp\\htdocs...', 122, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(122): mysqli->prepare('CALL STP_ErrLog...')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  thrown D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:05 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:05 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:05 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:05 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:05 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:05 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:05 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:05 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:06 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:06 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:06 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:06 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:06 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:06 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:07 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:07 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:07 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:07 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::real_connect(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\database\drivers\mysqli\mysqli_driver.php 201
ERROR - 2021-01-24 21:49:07 --> Unable to connect to the database
ERROR - 2021-01-24 21:49:07 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(618): CI_Exceptions->log_exception('Warning', 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120)
#2 [internal function]: _error_handler(2, 'mysqli::__const...', 'D:\\xampp\\htdocs...', 120, Array)
#3 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(120): mysqli->__construct('localhost', 'root', 'root', 'tatabarcode')
#4 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#5 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#6 [internal function]: _exception_handler(Object(Error))
#7 {main}
  th D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::__construct(): (08004/1040): Too many connections D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 120
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:07 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#2 [internal function]: _exception_handler(Object(Error))
#3 {main}
  thrown D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:07 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:07 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#2 [internal function]: _exception_handler(Object(Error))
#3 {main}
  thrown D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:07 --> Severity: error --> Exception: Call to a member function bind_param() on null D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:07 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:08 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:08 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:08 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:08 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:08 --> Severity: Warning --> mysqli::prepare(): Couldn't fetch mysqli D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 122
ERROR - 2021-01-24 21:49:08 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#2 [internal function]: _exception_handler(Object(Error))
#3 {main}
  thrown D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:49:08 --> Severity: Error --> Uncaught Error: Call to a member function bind_param() on null in D:\xampp\htdocs\TataToyo\system\core\Exceptions.php:123
Stack trace:
#0 D:\xampp\htdocs\TataToyo\system\core\Exceptions.php(110): CI_Exceptions->sendErrInDB('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#1 D:\xampp\htdocs\TataToyo\system\core\Common.php(653): CI_Exceptions->log_exception('error', 'Exception: Call...', 'D:\\xampp\\htdocs...', 123)
#2 [internal function]: _exception_handler(Object(Error))
#3 {main}
  thrown D:\xampp\htdocs\TataToyo\system\core\Exceptions.php 123
ERROR - 2021-01-24 21:53:14 --> 404 Page Not Found: Sounds/sound6.ogg
ERROR - 2021-01-24 22:31:30 --> 404 Page Not Found: Consignee/1
